app/models.py

```diff
@@ -352,6 +352,21 @@ class OrganisationTypes(db.Model):
     annual_free_sms_fragment_limit = db.Column(db.BigInteger, nullable=False)
 
 
+class OrganisationPermission(db.Model):
+    __tablename__ = "organisation_permissions"
+    id = db.Column(UUID(as_uuid=True), primary_key=True, nullable=False, default=uuid.uuid4)
+
+    created_at = db.Column(db.DateTime, nullable=False, default=datetime.datetime.utcnow)
+    organisation = db.relationship("Organisation")
+    organisation_id = db.Column(UUID(as_uuid=True), db.ForeignKey("organisation.id"), nullable=False)
+    permission = db.Column(
+        db.Enum(*ORGANISATION_PERMISSION_LIST, name="organisation_permission_types"),
```

Comment:
Just for your awareness, there's a bit of history on Notify around the use of enums. We have some enums but in other places we use so called "type tables", see NotificationStatusType or OrganisationType.

This is because of some historical migration issues that were encountered when updating enums. I'm not sure of the exact historical reasons, but there are probably more type tables than enums.

There's a bit of low-level discussion around what we'd like to rationalise on going forwards. For now I'm happy for us to keep this as an enum but there may be other opinions on the team.